docs/book/state_machine.rst

```diff
@@ -15,15 +15,56 @@ States
 
 States of a state machine are defined as constants on the model of an entity that the state machine is controlling.
 
+How to configure states? Let's see on the example of our **Checkout** state machine.
```

Comment:
Is that "our" like Sylius's or "our" like the users? I might say `Let's checkout the example from the **Checkout** ..` but unsure of the context.